.github/workflows/dependency-graph.yml

```diff
@@ -1,13 +1,19 @@
 name: Update Dependency Graph
 on:
+  pull_request:
   push:
     branches:
       - main # default branch of the project
 jobs:
   dependency-graph:
     name: Update Dependency Graph
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Setup Java 11
```

Comment:
I think java 11 + 8 are [already installed in the VM](https://github.com/actions/runner-images/blob/ubuntu24/20241215.1/images/ubuntu/Ubuntu2404-Readme.md). They just changed the default java used to Java 17. So `scalacenter/sbt-dependency-submission` is picking up the default Java. I'm not sure how to change the default